tests/lib/rules/no-missing-require.js

```diff
@@ -116,21 +111,21 @@ ruleTester.run("no-missing-require", rule, {
         // resolvePaths
         {
             filename: fixture("test.js"),
-            code: "require('fixtures/no-missing/a');",
+            code: "require('./fixtures/no-missing/a');",
```

Comment:
This test was invalid because `fixtures` was not a module, but instead a relative path from `./tests`.